design/images/container.puml

```diff
@@ -11,7 +12,8 @@ System_Boundary(seedcase, "Seedcase instance for Data Resource") {
   Container(management, "User and Data Management Layer")
 
   Container_Boundary(backend, "Back end environment") {
-    ContainerDb(backend_database, "Database")
+    ContainerDb(backend_database_data, "Database for data")
+    ContainerDb(backend_database_metadata, "Database for metadata")
```

Comment:
Add "Relational" to these two so it matches the naming in the component-backend figure?